src/plugins/plugin.legend.js

```diff
@@ -56,6 +57,26 @@ export class Legend extends Element {
  		 */
     this._hoveredItem = null;
 
+    this.navigation = {
```

Comment:
Wouldn't it be more logical if this was undefined unless someone has its feature enabled?

Reply:
I agree. Now the object is initialized only when navigation is enabled.